hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationManager.java

```diff
@@ -1169,15 +1154,6 @@ public void setMaintenanceReplicaMinimum(int replicaCount) {
     )
     private boolean push = true;
 
-    @PostConstruct
-    public void validate() {
-      if (!(commandDeadlineFactor > 0) || (commandDeadlineFactor > 1)) {
-        throw new IllegalArgumentException("command.deadline.factor is set to "
-            + commandDeadlineFactor
-            + " and must be greater than 0 and less than equal to 1");
-      }
-    }
```

Comment:
Let's keep `validate()` and check that `datanodeTimeoutOffset < eventTimeout` (maybe even enforce some minimum difference).

Reply:
It turns out that `TestBlockDeletion` sets `hdds.scm.replication.event.timeout`, so it also needs to set `hdds.scm.replication.event.timeout.datanode.offset`:

https://github.com/apache/ozone/actions/runs/4500741901/jobs/7920269601?pr=4453#step:5:3559

Reply:
Ah, I found one more occurrence too, after grepping for event.timeout. Hopefully they are fixed now.